docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and Kubernetes infrastructure against security best practices to help you identify and remediate misconfigurations.
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between Elastic cloud and self-hosted deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
+
+* From {kib}, open the main menu and click on **Manage this deployment**.
```

Comment:
This is technically step 1, right? Shouldn't this be a number instead of a bullet? 
```suggestion
* From {kib}, open the main menu and click on **Manage this deployment**.
```